scripts/generate_python_docs.sh

```diff
@@ -67,7 +67,11 @@ else
     # Install the Python package and run the pydocgen tool for just the provider that
     # was requested.
     echo "Building Python docs for ${REPO_OVERRIDE}..."
-    PACKAGE="pulumi_${REPO_OVERRIDE}"
+    if [ "${REPO_OVERRIDE}" = "pulumi" ]; then
+      PACKAGE="${REPO_OVERRIDE}"
+    else
+      PACKAGE="pulumi_${REPO_OVERRIDE}"
+    fi
```

Comment:
Nit: Missing extra 2 spaces of indentation inside the if/else. Could also be simplified as the following:

```suggestion
    PACKAGE="pulumi"
    if [ "${REPO_OVERRIDE}" != "pulumi" ]; then
        PACKAGE="pulumi_${REPO_OVERRIDE}"
    fi
```